<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>701</width>
    <height>543</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Адресная книга</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/icons/icons/adress1.png</normaloff>:/icons/icons/adress1.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTreeWidget" name="twBook">
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <attribute name="headerVisible">
       <bool>true</bool>
      </attribute>
      <attribute name="headerDefaultSectionSize">
       <number>183</number>
      </attribute>
      <attribute name="headerMinimumSectionSize">
       <number>0</number>
      </attribute>
      <column>
       <property name="text">
        <string>id</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Имя</string>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Фамилия</string>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Телефон</string>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/icons/icons/tel.png</normaloff>:/icons/icons/tel.png</iconset>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Электронная почта</string>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/icons/icons/91848.png</normaloff>:/icons/icons/91848.png</iconset>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Заметки</string>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/icons/icons/3640873.png</normaloff>:/icons/icons/3640873.png</iconset>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="sbMainForm"/>
  <widget class="QMenuBar" name="mbMain">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>701</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Правка</string>
    </property>
    <addaction name="acAdd"/>
    <addaction name="acUpd"/>
    <addaction name="acDel"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>Адресная книга</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="acAdd"/>
   <addaction name="acUpd"/>
   <addaction name="acDel"/>
   <addaction name="separator"/>
  </widget>
  <action name="acAdd">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/icons/add.png</normaloff>:/icons/icons/add.png</iconset>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
   <property name="shortcut">
    <string>Ins</string>
   </property>
  </action>
  <action name="acUpd">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/icons/edit-icon.png</normaloff>:/icons/icons/edit-icon.png</iconset>
   </property>
   <property name="text">
    <string>Изменить</string>
   </property>
   <property name="shortcut">
    <string>Return</string>
   </property>
  </action>
  <action name="acDel">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/icons/delete-icon-1.png</normaloff>:/icons/icons/delete-icon-1.png</iconset>
   </property>
   <property name="text">
    <string>Удалить</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
